samples/application-conf/CRB.properties

```diff
@@ -0,0 +1,27 @@
+# Language properties used by language/CRB.groovy
```

Comment:
@odttlnt @M-DLB ,
I see that these definitions are part of `application-conf` and `build-conf`. What would be the best location to manage these property definitions? Can we propose just one? 

If necessary, we would need to split it up into central build config, like `crb_zrbLocation` in build-conf and application specifics into the application configuration dir.

Reply:
I think it should be as is. Primarily because it provides more flexibility to the developer. They can mix and match the parameters and values